packages/components/src/Chips/InternalChipDismissible/hooks/useInternalChipDismissible.ts

```diff
@@ -3,6 +3,34 @@ import sortBy from "lodash/sortBy";
 import { useLiveAnnounce } from "@jobber/hooks/useLiveAnnounce";
 import { InternalChipDismissibleProps } from "../InternalChipDismissibleTypes";
 
+/**
+ * Recursively finds a focusable element (div or button) in the specified direction
+ */
+function findFocusableElement(
+  element: Element | null,
+  direction: "previous" | "next",
+): HTMLElement | null {
+  if (!element) return null;
+
+  const nextElement =
+    direction === "previous"
+      ? element.previousElementSibling
+      : element.nextElementSibling;
+
+  if (!nextElement) return null;
+
+  // Check if element is a div or button
+  if (
+    nextElement instanceof HTMLElement &&
+    (nextElement.tagName === "DIV" || nextElement.tagName === "BUTTON")
```

Comment:
we're trying to avoid giving the `span classname="shadow-activator"` focus

the valid sibling elements within Chips are
- button
- div
- input

Chip can be either a `div` or `button` depending on if the top level `Chips` was passed an `onClick` or not which is kind of weird. it's always interactive, I don't really agree with making it a `div` when no `onClick` is passed in to the `Chips`.

this criteria for being a focusableElement is pretty dependent on the current markup. though I am ok with having it be so tightly coupled with `Chip` because we don't support any other children for Chips.